models/cross-spork_client_test.go

```diff
@@ -0,0 +1,53 @@
+package models
```

Comment:
Same here regarding the `-` & `_` in file name, and maybe moving to `services` package.